modules/is_master/manifests/init.pp

```diff
@@ -0,0 +1,97 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+class is_master inherits is_master::params {
+
+  # Create distribution path
+  file { [  "${products_dir}",
+            "${products_dir}/${product}" ]:
+    ensure => 'directory',
+  }
+
+  # Change the ownership of the installation directory to wso2 user & group
```

Comment:
The code block right after this does not `Change the ownership of the installation directory` as mentioned in the comment.

Reply:
https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b